plugins/window_size/linux/window_size_plugin.cc

```diff
@@ -15,6 +15,7 @@
 
 #include <flutter_linux/flutter_linux.h>
 #include <gtk/gtk.h>
+#include <string.h>
```

Comment:
This should be `<cstring>`, not `<string.h>`.

Reply:
Okay, thanks!